programs/token-2022/src/state/multisig.rs

```diff
@@ -0,0 +1,93 @@
+use pinocchio::{
```

Comment:
Let's import `size_of`:
```suggestion
use core::mem::size_of;
use pinocchio::{
```